README.md

```diff
@@ -122,15 +122,21 @@ search for alternatives if you are running a different OS.*
 
 We have two supported installation methods, `pyenv` and `nix`. Pick which you prefer and proceed to that section.
 
-#### Setup: Pyenv
+#### Setup: Pyenv and Pipenv
 
-* Run
+When setting up for the first time, before you run `direnv allow`, run
 
   ```shell
   make install_tools
```

Comment:
I had to update my `pyenv` with instructions from `make install_tools` because Python 3.9.6 was not in the available list of installs.

```shell
Installing python 3.9.6 ...
python-build: definition not found: 3.9.6

See all available versions with `pyenv install --list'.

If the version you need is missing, try upgrading pyenv:

  cd /Users/duncan/.pyenv/plugins/python-build/../.. && git pull && cd -
failed

```

Reply:
Oh whoops! What version of python should we be using?

Reply:
I think that matches what's in the Dockerfile but more recent than when I updated pyenv.  I'm not sure how to address that in the brewfile if at all.

Reply:
Ah, I don't know either. I guess if you got it work, that's what counts?

Reply:
Weird, did you have `pyenv` previously? I wonder if we need to update the setup script to update `pyenv` if you already have it 🤔